model/project_ref.go

```diff
@@ -660,6 +660,10 @@ func (p *ProjectRef) Add(creator *user.DBUser) error {
 	if p.Id == "" {
 		p.Id = mgobson.NewObjectId().Hex()
 	}
+	// Default to adding the creator as the admin; the permissions themselves will be handled in the add function.
+	if creator != nil {
+		p.Admins = []string{creator.Id}
+	}
```

Comment:
```suggestion
	if creator != nil {
		p.Admins = append(p.Admins, creator.Id)
	}
```

Reply:
I was trying to decide whether to do this, and initially opted out since we don't actually add permissions for other admins in this function, but I think you're right this is still more correct. I'll verify that creator isn't already in this list as well.